collect_app/src/main/java/org/odk/collect/android/utilities/PermissionUtils.java

```diff
@@ -41,322 +42,287 @@
 
 public class PermissionUtils {
 
-    private PermissionUtils() {
+    /**
+     * Required for context and spawning of Dexter's activity that handles
+     * permission checking.
+     */
+    private final Activity activity;
+
+    public PermissionUtils(@NonNull Activity activity) {
+        this.activity = activity;
+    }
+
+    public static boolean checkIfStoragePermissionsGranted(Context context) {
+        return checkIfPermissionsGranted(context,
+                Manifest.permission.READ_EXTERNAL_STORAGE,
+                Manifest.permission.WRITE_EXTERNAL_STORAGE);
+    }
+
+    public static boolean checkIfCameraPermissionGranted(Context context) {
+        return checkIfPermissionsGranted(context, Manifest.permission.CAMERA);
+    }
+
+    public static boolean checkIfLocationPermissionsGranted(Context context) {
```

Comment:
I think those `checkIf`s can be replaced directly by `is` to make the names shorter, for example, `checkIfCameraAndRecordAudioPermissionsGranted` --> `isCameraAndRecordAudioPermissionsGranted`.

It's a tiny improvment, what do you think about that?

Reply:
Sounds good to me